pymeasure/instruments/tektronix/afg3152c.py

```diff
@@ -100,6 +104,24 @@ class AFG3152CChannel(Channel):
         """ Control the amplitude offset. It is always in Volt. (float)""",
     )
 
+    # phase degree should not be a query because it always returns rad
```

Comment:
Why don't you modify the response (with a [get_process](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/properties.html#processing-of-return-values))?